### PR TITLE
NEW - Ahmed Essam  - GSOD

### PR DESCRIPTION
**Pull request by Hanaffi**
I made new PR for all files better than last approach

### CODE REVIEW

**Comment by Hanaffi** [comment]
@huan @Rohitesh-Kumar-Jain

**Comment by Rohitesh-Kumar-Jain** [comment]
> @huan @Rohitesh-Kumar-Jain

We still have this error : 

```
# all asset files should be put into folder `/assets/YYYY/MM-slug-...-slug/` (slugs should be the same as the post)
not ok 13 "/assets/2021/05-gsod-2021-volunteering-proposal/gsod_s.webp" from "jekyll/_posts/2021-05-05-ahmed-essam-gsod-2021-volunteering-proposal.md" should be save to "assets/2021/05-ahmed-essam-gsod-2021-volunteering-proposal/"
  ---
    operator: fail
    at: Test.<anonymous> (/home/runner/work/wechaty.js.org/wechaty.js.org/tests/jekyll/asset-file-location.spec.ts:58:11)
    stack: |-
      Error: "/assets/2021/05-gsod-2021-volunteering-proposal/gsod_s.webp" from "jekyll/_posts/2021-05-05-ahmed-essam-gsod-2021-volunteering-proposal.md" should be save to "assets/2021/05-ahmed-essam-gsod-2021-volunteering-proposal/"
          at Test.assert [as _assert] (/home/runner/work/wechaty.js.org/wechaty.js.org/node_modules/tape/lib/test.js:228:54)
          at Test.bound [as _assert] (/home/runner/work/wechaty.js.org/wechaty.js.org/node_modules/tape/lib/test.js:80:32)
          at Test.fail (/home/runner/work/wechaty.js.org/wechaty.js.org/node_modules/tape/lib/test.js:322:10)
          at Test.bound [as fail] (/home/runner/work/wechaty.js.org/wechaty.js.org/node_modules/tape/lib/test.js:80:32)
          at Test.<anonymous> (/home/runner/work/wechaty.js.org/wechaty.js.org/tests/jekyll/asset-file-location.spec.ts:58:11)
  ...
```

Just fix this and we are good to go

**Comment by Hanaffi** [comment]
@Rohitesh-Kumar-Jain  Last check :( ?

**Comment by Rohitesh-Kumar-Jain** [comment]
> @Rohitesh-Kumar-Jain Last check :( ?

don't worry, I can check more : )

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Congratulations CI turned green 🎉 🥳

**Comment by Rohitesh-Kumar-Jain** [comment]
ping @huan

**Comment by Hanaffi** [comment]
@huan  Hey Huan, Can you merge? Its finally green :)

**Comment by Hanaffi** [comment]
@huan Can you merge please?

File: jekyll/_posts/2021-05-05-ahmed-essam-gsod-2021-volunteering-proposal.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Google Season of Docs 2021 Proposal - Volunteering"
+author: ahmed-essam
+categories: gsod
+tags:
+  - docs
+  - volunteering
+  - gsod
+  - proposal
+image: /assets/2021/05-ahmed-essam-gsod-2021-volunteering-proposal/gsod_s.webp
```
Comment (lijiarui) [inline jekyll/_posts/2021-05-05-ahmed-essam-gsod-2021-volunteering-proposal.md:10]: Please follow our file name rule:
> Create a markdown file in jekyll/_posts folder. The file name should follow the format YYYY-MM-DD-your-blog-slug.md. For example 2016-12-03-welcome-to-wechaty.md

Please change `gsod_s.webp` to `gsod-s.webp`

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
hi @Rohitesh-Kumar-Jain As GSoD Project PR Review Rules in https://github.com/wechaty/wechaty.js.org/issues/1045

This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.

**Comment by lijiarui** [comment]
Hi @Hanaffi , Since the `.webp` format has lots of advantages and the [safari has supported it last year](https://www.macrumors.com/2020/06/22/webp-safari-14/) which means all the modern browsers will be able to compatible with it.

@huan has just added a new unit test for enforcing the image format: the CI will not be passed if we are using other image formats than the perfect .webp.

Please make CI green to adjust the new rule, see https://github.com/wechaty/wechaty.js.org/issues/1035

**Comment by Rohitesh-Kumar-Jain** [comment]
Hi @Hanaffi , would you like to push more commits to turning the CI green?

**Review by huan — CHANGES_REQUESTED** [review]
Please check the CI and make sure it turns green before we can continue reviewing this PR, thanks!

See: https://github.com/wechaty/wechaty.js.org/pull/898/checks?check_run_id=3035429136#step:8:65

**Comment by lijiarui** [comment]
ping @Hanaffi

**Comment by lijiarui** [comment]
ping @Hanaffi

**Comment by Rohitesh-Kumar-Jain** [comment]
@Hanaffi feel free to reopen the PR, and push more commits to it :)

**Pull request closed without merging**